janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/rest/RestClientSetupTest.java

```diff
@@ -0,0 +1,562 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.es.rest;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.*;
```

Comment:
Can you put exact import instead of using * import ?

Reply:
Not to reopen the eternal debate...but

```
~/git/janusgraph nik$ find . -name \*.java | xargs egrep ^import | grep -v generated-sources | grep '\*' | wc -l
     400
```

I think you should probably set the rules for the coding style somewhere. And set up checkstyle plugin, or a similar one. From my personal experience, I always preferred the exact imports for the code but specifically for the unit tests and specifically for Mockito and JUnit I was making the exception.  Tests frequently change and people need new assertXYZ() or new method from Mockito - and this messes the imports on every commit. Not a huge deal, of course, they are for the compiler, not for the humans :)